src/Effects/EffectLFO.cpp

```diff
@@ -50,13 +51,38 @@ void EffectLFO::updateparams(void)
     lfornd = Prandomness / 127.0f;
     lfornd = (lfornd > 1.0f) ? 1.0f : lfornd;
 
-    if(PLFOtype > 1)
-        PLFOtype = 1;  //this has to be updated if more lfo's are added
+    if(PLFOtype > 2)
+        PLFOtype = 0;  //this has to be updated if more lfo's are added
     lfotype = PLFOtype;
     xr      = xl + (Pstereo - 64.0f) / 127.0f + 1.0f;
     xr      -= floorf(xr);
 }
 
+float EffectLFO::biquad(float input)
+{
+    float output;
```

Comment:
I'd move this to the first instance that the value is initialized

Reply:
ok